src/components/RevatureWorkExperience.tsx

```diff
@@ -66,7 +67,7 @@ const RevatureWorkExperience = () => {
     // Get data from data base
     //***********************************************************/
     const getData = async () => {
-        axios.get("http://3.236.213.150:8081/workexperience/portfolio/" + cookies['portfolio'].id)
+        axios.get(url +"/workexperience/portfolio/" + cookies['portfolio'].id)
```

Comment:
@ddinh please store `url + "/workexperience"` to a variable and pass here to minimize code duplication throughout this file